_source/logzio_collections/_log-sources/kubernetes.md

```diff
@@ -216,16 +216,57 @@ see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-
 
 </div>
 
+
+### Disabling systemd input
+
+To suppress Fluentd system messages, set the `FLUENTD_SYSTEMD_CONF` environment variable to `disable` in your Kubernetes environment.
+
+
+
+
 </div>
 <!-- tab:end -->
 
 
 <!-- tab:start -->
-<div id="disable">
+<div id="multiline">
 
-### Disabling systemd input
+### How to concatenate multiline logs
+
+
+If your original logs span multiple lines, you may find that they arrive in your Logz.io account split into several partial logs. You can use a Fluentd filter plugin to concatenate multiline logs and avoid having them separated into multiple log documents.
+
+For a full list of configuration options for `fluent-plugin-concat`, see the [GitHub project documentation](https://github.com/fluent-plugins-nursery/fluent-plugin-concat).
+
+
+###### Example
+
+The following is an example of a multiline log sent from a deployment on a k8s cluster:
+
+```shell
+2021-02-08 09:37:51,031 - errorLogger - ERROR - Traceback (most recent call last):
+File "./code.py", line 25, in my_func
+1/0
+ZeroDivisionError: division by zero
+```
```

Comment:
This example also applies to Filebeat, is there a reason it's only appears on Fluentd?